src/triggers/aws_lambda.js

```diff
@@ -115,6 +115,13 @@ function createSQSTrigger(event, trigger) {
     }, {
         'Message Body': event.Records[0].body,
     });
+    const messageBody = JSON.parse(event.Records[0].body || {});
```

Comment:
A comment is really needed (explain that you are extracting the data for the case for SQS task in step functions and so on)

Reply:
added.